.github/workflows/README.md

```diff
@@ -101,6 +120,10 @@ If you run into this issue, you can either:
 2) Upload secrets to your repo mirroring the secrets used in the main Beam repo.
 3) Wait until the changes are merged into Beam to test (this should only be done rarely).
 
+Additionally, as mentioned above your fork likely will not have self-hosted runners set up.
```

Comment:
same here, my understanding this section assumes branches on master.

Reply:
Same as above, lets consolidate to that thread